pycoshark/mongomodels.py

```diff
@@ -729,7 +729,7 @@ class Issue(Document):
     fix_versions = ListField(StringField())
     assignee_id = ObjectIdField()
     issue_links = ListField(DictField())
-    parent_issue_id = ObjectIdField()
+    parent_issue_id = StringField()
```

Comment:
Why this change? The _id should point to a Issue.id which would make ObjectIdField more suitable.

Reply:
Because we are now storing the issue external id not the mongo id. I can change the name to reflect the value or we can discuss how we can store the mongo id in the new system

Reply:
Then I would rename it. The parent_issue_id would make me expect a collection unique identifier analogous to issue_id which is not necessarily the case for external_id. I think we can call it parent_issue_external_id to keep it consistent.

Reply:
I added link_issue function to IssueShark so now we can store issued_id. I put the field type as Dynamic field because first we store the external id an then in link_issue we replace it with object id.